guides/source/install_ruby_on_rails.md

```diff
@@ -7,8 +7,16 @@ This guide will walk you through installing the Ruby programming language and th
 
 While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
 
+You can also use:
+- [ASDF](https://asdf-vm.com)
+- [rbenv](https://rbenv.org)
+- [chruby](https://github.com/postmodern/chruby)
```

Comment:
Seems `chruby` is not maintaining properly.

Reply:
I’m not well versed enough in the specific state of chruby to know for sure